lms/static/js/fixtures/commerce/checkout_receipt.html

```diff
@@ -0,0 +1,27 @@
+<div id="error-container" class="hidden">
+    <div id="error" class="wrapper-msg wrapper-msg-activate">
+        <div class=" msg msg-activate">
+            <i class="msg-icon icon fa fa-exclamation-triangle" aria-hidden="true"></i>
+
+            <div class="msg-content">
+                <h3 class="title">
+                    Error
+                </h3>
+                <div class="copy">
+                    <p>dummy error text</p>
+                    <br/>
+                </div>
+            </div>
+        </div>
+    </div>
+</div>
+
+<div class="container">
+    <section class="wrapper carousel">
+        <div id="receipt-container" class="pay-and-verify hidden" data-is-payment-complete='True'
+             data-platform-name='edx-platform' data-verified='True' data-username='user-1'>
+            <h1>${_("Loading Order Data...")}</h1>
```

Comment:
Does the header of the page already contain an h1? If so this should be updated to be an h2.